appsec/appsec.go

```diff
@@ -30,3 +33,92 @@ func MonitorParsedHTTPBody(ctx context.Context, body interface{}) {
 	}
 	// bonus: use sync.Once to log a debug message once if AppSec is disabled
 }
+
+// TrackUserLoginSuccessEvent sets a successful user login event, with the given
+// user id and optional metadata, as service entry span tags. It also calls
+// tracer.SetUser() to set the currently identified user, along the given
+// tracer.UserMonitoringOption options.
+// The service entry span is obtained through the given Go context which should
+// contain the currently running span. This function does nothing when no span
+// is found in the given Go context, along with an error message.
```

Comment:
I think something like this may be clearer. WDYT?
```suggestion
// is found in the given Go context and logs an error message instead.
```